task3.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void printToConsole(char character, int counter)
+{
+    if (counter == 1)
+    {
+        printf("%c", character);
+    }
+    else
+    {
+        printf("%d%c", counter, character);
+    }
+}
+
+int main(void)
+{
+    FILE *file = fopen("input.txt", "r");
+    if (file == NULL)
+    {
+        printf("File not found!\n");
+        return -1;
+    }
+
+    char firstCharacter = fgetc(file);
+    char secondCharacter = 0;
```

Comment:
```suggestion
    char firstCharacterOfGroup = fgetc(file);
    char currentCharacter = 0;
```
Вот так понятнее будет, что за переменные.